.github/workflows/release.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
           echo "sha_short=$(git rev-parse --short HEAD)" >> $GITHUB_ENV
 
       - name: Login to DockerHub
-        uses: docker/login-action@v3
+        uses: docker/login-action@343f7c4344506bcbf9b4de18042ae17996df046d  # v3.0.0
```

Comment:
why changes all these? will depbot update this later?

Reply:
its more secure to reference them by has rather than by version - but we still need the version for dependabot so its added as a comment